src/main/java/com/example/bookshop/repository/CategoryRepository.java

```diff
@@ -0,0 +1,9 @@
+package com.example.bookshop.repository;
+
+import com.example.bookshop.model.Category;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.JpaSpecificationExecutor;
+
+public interface CategoryRepository extends JpaRepository<Category, Long>,
+        JpaSpecificationExecutor<Category> {
```

Comment:
```suggestion
         {
```
Why do you need a specification executor here?